app/MailTemplateFixtures/VariantSettingsGenerator.php

```diff
@@ -19,7 +19,7 @@ public function __construct(
 	 */
 	public function getTemplateSettings(): iterable {
 		foreach ( $this->variants as $variant ) {
-			$templateData = array_merge( $this->templateData, $variant->additionalTemplateData );
+			$templateData = array_merge_recursive( $this->templateData, $variant->additionalTemplateData );
```

Comment:
Run the CLI command with array_merge and you'll see it fires errors because the donation ID is being overwritten by array_merge. We could keep it and add the donation id to the variant settings?